.github/workflows/ci.yml

```
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [18.x, 20.x]
+        node-version: [18.x, 19.x, 20.x, 21.x, 22.x, 23.x]
```

node 19 is not supported by yo.
```suggestion
        node-version: [18.x, 20.x, 21.x, 22.x, 23.x]
```